mods/fire/init.lua

```diff
@@ -249,21 +259,16 @@ else
 		end,
 	})
 
-	-- Remove basic flames and flammable nodes
+	-- Remove flammable nodes
 
 	minetest.register_abm({
 		nodenames = {"fire:basic_flame"},
```

Comment:
Since this ABM is only for removing flammable nodes next to flames, we can add 'neighbors = "group:flammable"' here as it no longer needs to operate on isolated flames.